synapse/config/key.py

```diff
@@ -41,6 +41,15 @@
 you are *sure* you want to do this, set 'accept_keys_insecurely' on the
 keyserver configuration."""
 
+RELYING_ON_MATRIX_KEY_ERROR = """\
```

Comment:
```suggestion
RELYING_ON_MATRIX_KEY_ERROR = """
```

Reply:
nope, this is deliberate. I don't want the leading newline.